great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -59,6 +59,7 @@ class _FilePathDataAsset(DataAsset):
         "batching_regex",  # file_path argument
         "kwargs",  # kwargs need to be unpacked and passed separately
         "batch_metadata",
+        "connect_options",
```

Comment:
@Kilo59 This is needed -- otherwise, instantiating an ExecutionEngine says that "options is not recognized by __init__().